### ISSUE TITLE
Keyboard question

### ISSUE THREAD

**Issue by sgriffin916** [tbleckert/react-select-search#26]
When I type in a value in the search list on the demo page, should I be able to use the down arrow key to get to various items?  For instance, on the country one, if I type "Fr", I would expect to be able to hit the down arrow key and get to French Guiana.  Right now, the only way I see to select it is typing enough that it is the first item on the list or using the mouse.

Thanks!

**Comment by jonathancochran** [comment]
Seconding this. 

It seems like this used to work but I'm experiencing the same issue in testing.

**Comment by jonathancochran** [comment]
It turns out that the keyboard arrows works when you have items in groups. Probably a bug but you can work around it by putting your data in this format: 
```
const options = [
    {
        type: 'group',
        items: [
            {name: 'Spanish', value: 'es'},
            {name: 'English', value: 'en'},
        ]
    },
];
```

**Comment by sgriffin916** [comment]
Yep, that worked for me.  Thanks for the tip!

**Comment by tbleckert** [comment]
@jonathancochran Nice! Probably got broken when I implemented the grouping functionality. @sgriffin916 I'll reopen this if it's ok so we can fix the bug. Also, thanks for reporting!

**Comment by tbleckert** [comment]
Fixed in #32
